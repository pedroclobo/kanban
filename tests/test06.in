t 1 uma tarefa com uma longa e pormenorizada descricao
l
q
