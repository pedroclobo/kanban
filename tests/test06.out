task 1
1 TO DO #1 uma tarefa com uma longa e pormenorizada descricao
